simde/simde-math.h

```diff
@@ -706,6 +706,16 @@ simde_math_fpclassify(double v) {
   #endif
 #endif
 
+#if !defined(simde_math_signbit)
+  #if SIMDE_MATH_BUILTIN_LIBM(signbit)
+    #define simde_math_signbit(x) __builtin_signbit(x)
```

Comment:
In old versions of clang & GCC, the `__builtin_signbit` function is only for doubles.

clang-8, no error:  https://nemequ.semaphoreci.com/jobs/d4bfe35a-c9d1-4be5-8278-8afa5dfcf0f3
clang-6.0 error: https://nemequ.semaphoreci.com/jobs/4df6cec3-b65b-4d31-b48b-58a436667ecf
clang-4 error: https://nemequ.semaphoreci.com/jobs/f94a6879-6b34-457c-a82a-ca1f1a7c8ac0

How would you like to handle this? Maybe and `#if def` to only add a cast to `double` for these old compilers?

https://android.googlesource.com/platform/prebuilts/clang/darwin-x86/sdk/3.5/+/refs/heads/studio-1.3-release/include/clang/Basic/Builtins.def#374
For Clang, appears this was fixed in version 7.0 https://github.com/llvm/llvm-project/commit/dd0e2b01ae58f3b5c9cf8adfd016c9d239458955

Looks like GCC fixed this sometime after GCC 4.4.0 https://gcc.gnu.org/bugzilla/show_bug.cgi?id=36757 ; so I don't think we need GCC specific handling here.

Reply:
I'm testing https://github.com/simd-everywhere/simde/commit/c22e7c5a4d3daff8a02b3dff3c0ccebce8641ddd now